WordPress/Classes/ViewRelated/Views/WPRichText/WPTextAttachment.swift

```diff
@@ -37,8 +38,8 @@ open class WPTextAttachment: NSTextAttachment {
         self.identifier = identifier
         self.tagName = tagName
         self.src = src
-
-        super.init(data: nil, ofType: nil)
+        let img = UIImage.init(color: UIColor.white)
+        super.init(data: img!.pngData(), ofType: kUTTypePNG as String)
```

Comment:
We could condense this all to:

`super.init(data: UIImage(color: .basicBackground).pngData(), ofType: kUTTypePNG as String)`

(Also changed the color there to handle dark mode)

Reply:
Thanks for the suggestion. Much nicer than what my tired brain was thinking yesterday evening. :+1: